ArduPlane/Parameters.h

```diff
@@ -345,8 +345,11 @@ class Parameters {
         k_param_pidWheelSteer, // unused
 
         k_param_mixing_offset,
-        k_param_dspoiler_rud_rate,
-
+        
+        // 250: ArduSoar parameters
+        k_param_soaring_controller,
```

Comment:
these should be in g2. I see you have a commit that says it causes a crash? I'd be happy to investigate if you like